logstash-output-dogstatsd.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # Files
-  s.files = `git ls-files`.split($\)+::Dir.glob('vendor/*')
+  s.files = Dir['lib/**/*','spec/**/*','vendor/**/*','*.gemspec','*.md','CONTRIBUTORS','Gemfile','LICENSE','NOTICE.TXT']
```

Comment:
Thanks for this, though would you mind removing `spec/**/*` and `vendor/**/*` from this list while you're here since those directories either aren't necessary or don't exist?